src/peerstate.rs

```diff
@@ -521,6 +490,170 @@ impl Peerstate {
             false
         }
     }
+
+    async fn handle_change(
+        &self,
+        context: &Context,
+        timestamp: i64,
+        change: ChatsChange,
+    ) -> Result<(), anyhow::Error> {
+        if context.is_self_addr(&self.addr).await? {
```

Comment:
function documentation is needed. It's a central "where things actuallly happen" part of AEAP.  A better function name would thus be "emit_aeap_info_messages" or so.

Reply:
we can't name it `emit_aeap_info_messages()` since the same function also emits info messages if the peerstate's fingerprint changed. Except if we revert 7b5b27da and accept the code duplication between the two functions.

Reply:
maybe handle_setup_change or so. or leave it as is if you don't like it.

Reply:
Done